src/test/java/org/jenkinsci/plugins/workflow/DSLTest.java

```diff
@@ -148,6 +148,30 @@ public void nonexistentFunctions() throws Exception {
         r.assertLogContains("constructible with compass and straightedge", b);
     }
 
+    /**
+     * Tests the ability to execute a step with an unnamed monomorphic describable argument.
+     */
+    @Issue("JENKINS-29711")
+    @Test
+    public void monomorphic() throws Exception {
+        WorkflowJob p = r.jenkins.createProject(WorkflowJob.class, "mon");
+        p.setDefinition(new CpsFlowDefinition("monomorphStep([firstArg:'one', secondArg:'two'])", true));
```

Comment:
So, since IIUC this is what was working before, why is the patch to `DSL` still here? In other words, if you revert that part, does any current test fail?